ledger-tool/src/main.rs

```diff
@@ -1817,11 +1821,11 @@ fn main() {
                         working_bank.print_accounts_stats();
                     }
                     if print_bank_hash {
-                        println!(
-                            "Bank hash for slot {}: {}",
-                            working_bank.slot(),
-                            working_bank.hash()
-                        );
+                        let slot_bank_hash = SlotBankHash {
+                            slot: working_bank.slot(),
+                            hash: working_bank.hash().to_string(),
+                        };
+                        println!("{}", output_format.formatted_string(&slot_bank_hash));
```

Comment:
Have you given any thought to how to handle once `shred_version` is also optionally printed? If a single, flat map, can be handled in later PR. If a nested map (eg. `{"bankHash": {"slot": .. }, "shredVersion": .. }`), might be best to set that up here.

Imo, flat map would be better, if it makes sense.

Reply:
> Have you given any thought to how to handle once shred_version is also optionally printed?

I gave a little thought, but couldn't think of a solution I loved. Conceptually, I think the types for `shred-version` and `bank-hash` could/should be separate. But you're absolutely right that if I implement `shred-version` in an identical manner, we'd print this non-ideal output:
```
{
  "bankHash": ...
}
{
  "shredVersion": ...
}
```
I probably prefer the flat-map as well, will have to take a sec to think about how to most cleanly do that